netket/experimental/observable/variance/variance_operator.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, operator: AbstractOperator, use_fast_estimator: bool = False)
 
         Args:
             operator: The operator for which the variance is to be computed.
-            use_fast_estimator: If False, uses the local estimator of the squared operator `O^2` for variance computation (defaults to False).
+            use_Oloc_squared: If False, uses the local estimator of the squared operator `O^2` for variance computation (defaults to False).
```

Comment:
Please change to something like  '(Defaults False) if True, uses the fast estimator obtained by squaring the local estimator O^loc. If False, uses the straightforward estimator of (O@O)_loc which is quadratically more expensive. The fast estimator can sometimes lead to worse quality results.

Reply:
Biensûr Filippo